modules/installation-and-upgrade/pages/container-deployment/suma/sle-micro-deployment.adoc

```diff
@@ -28,30 +29,32 @@ To guarantee that clients can resolve the {productname} {productnumber} domain n
 
 == {micro} {microversion} Installation
 
-Get a copy of the {micro} {microversion} install media. You can begin installation on either a virtual machine or on a physical server. 
+Get a copy of the {micro} {microversion} installation media. You can begin the installation on either a virtual machine or on a physical server.
 
 
 .Procedure: {micro} {microversion} installation
-. Begin installation from media
+. Begin the installation from the installation media.
 
 . Adjust Keyboard and language. Click the [systemitem]``checkbox`` to accept the License Agreement.
 
 . Click [systemitem]``Next`` to continue.
 
-. Select your registration method. For this example we will register the sever with {scc}.
+. Select your registration method. For this example we will register the sever with the {scc}.
 
-. Enter your {scc} Email address. 
+. Enter your {scc} email address.
```

Comment:
```suggestion
. Enter your {scc} E-mail Address.
```

Reply:
From what I remember, in English, `email` is not written with a dash between the `e` and `mail` like we do it in German. Do you know more here, @jcayouette?

Reply:
It should be: "Enter your {SCC} email address."

Reply:
Thank you, Joseph. Then I leave my changes.